Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -74,8 +74,24 @@ await context.Configuration.Events.AfterNavigation
 
     public static string GetCurrentAbsolutePath(this UITestContext context) => context.GetCurrentUri().AbsolutePath;
 
-    public static Uri GetAbsoluteUri(this UITestContext context, string relativeUrl) =>
-        new(context.Scope.BaseUri, relativeUrl.TrimStart('/'));
+    public static Uri GetAbsoluteUri(this UITestContext context, string relativeUrl)
+    {
+        var localPath = context.Scope.BaseUri.LocalPath;
+        var splitRelativeUrl = relativeUrl.TrimStart('/').Split('?');
+        var query = splitRelativeUrl.Length > 1 ? splitRelativeUrl[1] : string.Empty;
+        var path = splitRelativeUrl[0];
+        var uriBuilder = new UriBuilder
+        {
+            Scheme = context.Scope.BaseUri.Scheme,
+            Host = context.Scope.BaseUri.Host,
+            Port = context.Scope.BaseUri.Port,
+            Path = string.IsNullOrEmpty(localPath) || localPath == "/"
+                ? path
+                : localPath.TrimStart('/').TrimEnd('/') + "/" + path,
+            Query = query,
+        };
+        return uriBuilder.Uri;
```

Comment:
Why is this logic necessary? Explain it in a comment.

But is it really necessary? `RequestUrlPrefix` is only for tenants, and when calling `UITestContext.SwitchCurrentTenant()`, which should happen when switching between tenants, the `BaseUri` will be suffixed with it.

Reply:
Yes, it will be suffixed with it but `new Uri(Uri, string)` does the following:
Base uri: `https://tenants-staging.dotnest.net/editeddotnesttenantscoretenantmanagementremoteuitest`
relativeUrl: `/login`
Output: `https://tenants-staging.dotnest.net/login`

So the path will be deleted from the base URI.

Let me know if you would prefer 
`new Uri(context.Scope.BaseUri.OriginalString + "/" + relativeUrl.TrimStart('/'))` 
instead.

Reply:
Then rather the base URI should be suffixed with `/` if that's still accepted by OC for `RequestUrlPrefix`es (when it's just a host, it's always suffixed), or if not, then what you write.